docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -1,301 +1,166 @@
-```
-
-We now construct the reaction network. The basic types of arrows and predefined
-rate laws one can use are discussed in detail within the tutorial, [The Reaction
-DSL](@ref dsl_description). Here, we use a mix of first order, zero order, and repressive Hill
-function rate laws. Note, $\varnothing$ corresponds to the empty state, and is
-used for zeroth order production and first order degradation reactions:
-```@example tut1
-repressilator = @reaction_network Repressilator begin
-    hillr(P₃,α,K,n), ∅ --> m₁
-    hillr(P₁,α,K,n), ∅ --> m₂
-    hillr(P₂,α,K,n), ∅ --> m₃
-    (δ,γ), m₁ <--> ∅
-    (δ,γ), m₂ <--> ∅
-    (δ,γ), m₃ <--> ∅
-    β, m₁ --> m₁ + P₁
-    β, m₂ --> m₂ + P₂
-    β, m₃ --> m₃ + P₃
-    μ, P₁ --> ∅
-    μ, P₂ --> ∅
-    μ, P₃ --> ∅
-end
-show(stdout, MIME"text/plain"(), repressilator) # hide
-```
-showing that we've created a new network model named `Repressilator` with the
-listed chemical species and unknowns. [`@reaction_network`](@ref) returns a
-[`ReactionSystem`](@ref), which we saved in the `repressilator` variable. It can
-be converted to a variety of other mathematical models represented as
-`ModelingToolkit.AbstractSystem`s, or analyzed in various ways using the
-[Catalyst.jl API](@ref). For example, to see the chemical species, parameters,
-and reactions we can use
-```@example tut1
-species(repressilator)
-```
-```@example tut1
-parameters(repressilator)
-```
-and
-```@example tut1
-reactions(repressilator)
-```
-We can also use Latexify to see the corresponding reactions in Latex, which shows what
-the `hillr` terms mathematically correspond to
+Before starting this tutorial, we must (unless this has already been done) [install and activate](@ref ref) the Catalyst package:
 ```julia
-latexify(repressilator)
+using Pkg
+Pkg.add("Catalyst")
+using Catalyst
+```
+
+## [Creating basic Catalyst models](@id introduction_to_catalyst_model_creation)
+Catalyst exports the [`@reaction_network`](@ref) [macro]([macro](https://docs.julialang.org/en/v1/manual/metaprogramming/#man-macros)), which provides the main way of creating CRN models (alternative approaches are described [later](@ref ref)). This is followed by a `begin ... end` block which encapsulates the reactions of the system. E.g here we create a simple [birth-death model](@ref ref) (where a single species $X$ is both produced and degraded):
+```@example intro_1
+using Catalyst # hide
+bd_model = @reaction_network begin
+ p, 0 --> X
+ d, X --> 0
+end
 ```
-```@example tut1
-repressilator #hide
+Next, we create a simple [Michaelis-Menten enzyme kinetics model](@ref ref) (where an enzyme, $E$ converts a substrate, $S$, to a product, $P$):
+```@example intro_1
+mm_model = @reaction_network begin
+ kB, S + E --> SE
+ kD, SE --> S + E
+ kP, SE --> P + E
+end
 ```
-Assuming [Graphviz](https://graphviz.org/) is installed and command line
-accessible, within a Jupyter notebook we can also graph the reaction network by
-```julia
-g = Graph(repressilator)
```

Comment:
Last time we discussed I think we said that it made sense to move the graph visualisation out of the intro tutorial, and instead have a specific section.

Reply:
(the dedicated model visualisation doc page is written and should be merged as soon as the tests can be run again)